assets/ui/taskSelectionScreen.ui

```diff
@@ -1,75 +1,136 @@
+                                "use-content-height": true,
+                                "position-right": {
+                                    "offset": 1
                                 },
-                                {
-                                    "type" : "ColumnLayout",
-                                    "contents" : [
-                                        {
-                                            "type" : "UIButton",
-                                            "text" : "Buildings",
-                                            "id"   : "buildingsTabButton",
-                                            "family" : "title"
-                                        }
-                                    ]
-                                }
+                                "position-horizontal-top": {
+                                    "offset": 1
+                                },
+                                "width": 30,
+                                "height": 30
+                            }
+                        },
+                        {
+                            "type" : "UILabel",
+                            "text" : "PICK YOUR MAGIC",
+                            "id"   : "environmentTaskInfoLabel",
 
-                            ]
+                            "layoutInfo": {
+                                "use-content-width": true,
+                                "position-horizontal-center": {
+
+                                },
+                                "height": 30
+                            }
                         },
                         {
                             "type" : "RowLayout",
+                            "id" : "options",
                             "contents" : [
                                 {
-                                    "type" : "UIList",
-                                    "id" : "taskSelectionScreenList"
+                                    "type" : "UIButton",
+                                    "text" : "Tasks",
+                                    "id"   : "tasksTabButton",
+                                    "family" : "title"
+
+
+                                },
+                                {
+                                    "type" : "UIButton",
+                                    "text" : "Buildings",
+                                    "id"   : "buildingsTabButton",
+                                    "family" : "title"
+
                                 }
-                            ]
+                            ],
+                            "layoutInfo": {
+                                "use-content-height": true,
+                                "position-top": {
+                                  "offset": 32
+                                },
+                                "position-horizontal-center": {
+                                  "offset": -5
```

Comment:
Does this cause the buttons to float out to the left? Maybe we're looking for a padding/margin here instead of an offset?

Reply:
I didn't try padding/margin, I'll look into it, alright!

Reply:
I had a quick look at this, and I did not find anything like padding or margin. However, just removing the offset does the job of moving the two buttons straight in the center. 

```suggestion
```

![image](https://user-images.githubusercontent.com/1448874/83975456-651b5c80-a8f4-11ea-8faa-e8659e19dc35.png)

Reply:
However, we can fake some margins by using `horizontalSpacing` property of the `RowLayout` and adjusting the overall `width` to get some gaps (if desired):

![image](https://user-images.githubusercontent.com/1448874/83975525-aca1e880-a8f4-11ea-842c-0764d9ab3ff1.png)